root/release/edit/information.tt

```diff
@@ -51,6 +51,39 @@
         </td>
       </tr>
 
+      <tr data-bind="if: $root.loadingDuplicateRGs()">
+        <td></td>
+        <td colspan="2">
+          <div class="duplicate-rgs-loading loading-message">
+            [% l('Checking existing release groups…') %]
+          </div>
+        </td>
+      </tr>
+
+      <tr data-bind="if: $root.duplicateRGs().length > 0">
+        <td></td>
+        <td colspan="2">
+          <div class="duplicate-rgs-label">
+            [% add_colon(l('Existing release groups with similar names')) %]
+          </div>
+          <div class="duplicate-rgs-list">
+            <!-- ko foreach: $root.duplicateRGs -->
+              <label>
+                <input name="rg-selection" type="radio" data-change="selectReleaseGroup" />
```

Comment:
Just to mention it, clicking on one of the radio buttons selects the corresponding release group and makes the list disappear. Not sure if that feels too weird from a UX perspective. It seems to interact poorly with HTML's weird keyboard focus model for radio buttons, so I probably ought to make this behave differently.

Reply:
I've tweaked the logic to keep the radio buttons visible after one is selected, which I think feels better.